### PR TITLE
update: add links to api reference on @JvmDefaultWithCompatibility

### PR DESCRIPTION
**Pull request by PetrakovichVictoria**
The API reference was updated after the 1.6.20 release.

### CODE REVIEW

File: docs/topics/jvm/java-to-kotlin-interop.md [hunk 1]
```diff
@@ -338,7 +338,7 @@ public class BB8 implements Robot {
 
 If there are clients that use your Kotlin interfaces compiled without the `-Xjvm-default=all` option, then they may
 be binary-incompatible with the code compiled with this option. To avoid breaking the compatibility with such clients, 
-use the `-Xjvm-default=all` mode and mark interfaces with the `@JvmDefaultWithCompatibility` annotation. 
+use the `-Xjvm-default=all` mode and mark interfaces with the [`@JvmDefaultWithCompatibility` annotation](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-jvm-default-with-compatibility/).
```
Comment (koshachy) [inline docs/topics/jvm/java-to-kotlin-interop.md:341]:
```suggestion
use the `-Xjvm-default=all` mode and mark interfaces with the [`@JvmDefaultWithCompatibility`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-jvm-default-with-compatibility/) annotation. 
```

File: docs/topics/whatsnew1620.md [hunk 2]
```diff
@@ -131,7 +131,7 @@ Kotlin 1.6.20 introduces:
 
 ### New @JvmDefaultWithCompatibility annotation for interfaces
 
-Kotlin 1.6.20 introduces the [new annotation `@JvmDefaultWithCompatibility`](https://github.com/JetBrains/kotlin/blob/master/libraries/stdlib/jvm/src/kotlin/jvm/JvmDefault.kt#L53): use it along with the `-Xjvm-default=all` compiler option [to create the default method in JVM interface](java-to-kotlin-interop.md#default-methods-in-interfaces) for any non-abstract member in any Kotlin interface.
+Kotlin 1.6.20 introduces the [new annotation `@JvmDefaultWithCompatibility`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-jvm-default-with-compatibility/).: use it along with the `-Xjvm-default=all` compiler option [to create the default method in JVM interface](java-to-kotlin-interop.md#default-methods-in-interfaces) for any non-abstract member in any Kotlin interface.
```
Comment (koshachy) [inline docs/topics/whatsnew1620.md:134]:
```suggestion
Kotlin 1.6.20 introduces the new annotation [`@JvmDefaultWithCompatibility`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-jvm-default-with-compatibility/): use it along with the `-Xjvm-default=all` compiler option [to create the default method in JVM interface](java-to-kotlin-interop.md#default-methods-in-interfaces) for any non-abstract member in any Kotlin interface.
```

File: docs/topics/jvm/java-to-kotlin-interop.md (hunk 1 above)
Comment (PetrakovichVictoria) [reply docs/topics/jvm/java-to-kotlin-interop.md:341]: Don't you think including the word annotation into the link would be better from the SEO perspective?

Comment (koshachy) [reply docs/topics/jvm/java-to-kotlin-interop.md:341]: It is pretty similar to the case, when we provide links to particular functions. 
There is no reason to think about SEO: there is no competitive content, annotations with the same name on other sites, etc.

Actually, if users will search for this annotation, the most relevant page for him is this: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-jvm-default-with-compatibility/

**Pull request merged**